docs/source/features/sheet_management/search.rst

```diff
@@ -0,0 +1,139 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+##################
+ Search Operation
+##################
+
+This topic describes Graviti DataFrame search operations:
+
+   * :py:meth:`~graviti.dataframe.frame.query`
+   * :py:meth:`~graviti.dataframe.frame.appaly`
+
+*******
+ Query
+*******
```

Comment:
should we introduce Dataset preparations first, and then introduce Query examples and apply examples. And finally, we could introduce how to use them together